src/scenes/ProgressReports/EditForm/Steps/Media/MediaInfoForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, TextField } from '@mui/material';
+import {
+  ProgressReportMediaCategoryLabels,
+  ProgressReportMediaCategoryList,
+} from '~/api/schema/enumLists';
+import { labelFrom } from '~/common';
+import { Form, SelectField } from '~/components/form';
+import { minLength, required } from '~/components/form/validators';
+import { useCallback, useState } from 'react';
+import { ProgressReportMediaCategory } from '~/api/schema.graphql';
+
+// TODO type submitForm
+// @ts-ignore
+export const MediaInfoForm = ({
+  submitForm,
+  disabled,
+  initialValues,
+}: {
+  submitForm: any;
+  disabled: boolean;
+  initialValues: {
+    category: ProgressReportMediaCategory | null;
+    caption: string | null;
+  };
+}) => {
+  const [category, setCategory] = useState(initialValues.category);
+  const [caption, setCaption] = useState(initialValues.caption);
+  const updateMedia = useCallback(() => {
+    submitForm({ variables: {} });
+  }, []);
```

Comment:
The form abstraction handles all the state management, so we don't need to do this. Should just be able to pass up `onSubmit`.